addons/mail/wizard/mail_compose_message_views.xml

```diff
@@ -79,7 +79,7 @@
                         <button string="Send" attrs="{'invisible': [('is_log', '=', True)]}" name="action_send_mail" type="object" class="btn-primary o_mail_send" data-hotkey="q"/>
                         <button string="Log" attrs="{'invisible': [('is_log', '=', False)]}" name="action_send_mail" type="object" class="btn-primary" data-hotkey="q"/>
                         <button string="Cancel" class="btn-secondary" special="cancel" data-hotkey="z" />
-                        <button icon="fa-lg fa-save" type="object" groups="mail.group_mail_template_editor"
+                        <button icon="fa-lg fa-save" type="object"
```

Comment:
could be worth it to check the "mail composer mixin"

do we do the same logic for them ? (currently if you select a template, you won't be able to edit the body if you don't have the group, because it's hard to know if you changed the code or just if the web editor moved css properties)

so for them, we can either
- do not change them (which means that you won't be able to select a static template and edit it)
- same logic as for mail template (so you can edit the body as long as there's no code in it)

Reply:
There are two cases to distinguish:
 - the composer in single record mode, e.g. when expanding / hitting the full composer button in a message thread. In this case a regular user can always edit the content because it's pre-rendered and then not rendered anymore on change.
 - the composer in mass mode, e.g. when selecting multiple record in the list and hitting the "Send email" button in the action dropdown menu. In this case a regular user cannot edit the content (currently).

I think we can keep the current behavior. I do not think the second case is what our salesman were against, though I will confirm with them. 

But, now that I see that, the "Save as new template" for regular users in the second case doesn't really make sense, as they cannot edit it.
Therefore instead of simply removing the groups, I replaced it to an attrs which makes the button invisible if the user cannot edit the content.

Reply:
I guess it's ok for stable, if sales have no problem with that, I'll change that in master anyway